scripts/Package_BGA/bga.yml

```diff
@@ -1,3 +1,14 @@
+Fujitsu_WLP-15_2.28x3.092mm_Layout3x5_P0.4mm:
+  description: "WLP-15, 3x5 raster, 2.28x3.092mm package, pitch 0.4mm; http://www.fujitsu.com/global/documents/products/devices/semiconductor/fram/lineup/MB85RS1MT-DS501-00022-7v0-E.pdf"
+  pkg_width:   2.28
+  pkg_height:  3.092
+  pitch: 0.4
```

Comment:
Pitch is ok for x direction. But y direction would require 0.3mm. I don't think the script would support this right now.